frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -237,14 +247,56 @@ export const InstallPlansPage: React.FC<InstallPlansPageProps> = (props) => {
   );
 };
 
+const updateUser = (isOpenShift: boolean, user: UserKind): string => {
+  if (!isOpenShift) {
+    return authSvc.name();
+  }
+  return user.fullName || user.metadata?.name || '';
+};
+
+export const NeedInstallPlanPermissions = ({ installPlan }: NeedInstallPlanPermissions) => {
+  const isOpenShift = useFlag(FLAGS.OPENSHIFT);
+  const user = useSelector<RootState, string>(({ UI }) => UI.get('user'));
+
+  const [username, setUsername] = React.useState(updateUser(isOpenShift, user));
+
+  React.useEffect(() => {
+    setUsername(updateUser(isOpenShift, user));
+  }, [isOpenShift, user]);
+
+  const { t } = useTranslation();
+
+  const apiGroup = apiGroupForReference(referenceFor(installPlan));
+
+  return (
+    <Alert
+      variant="info"
+      isInline
+      title={t('olm~Missing sufficient privileges for manual InstallPlan approval')}
+    >
+      {t(
+        'olm~User "{{user}}" does not have permissions to patch resource InstallPlans in API group "{{apiGroup}}" in the namespace "{{namespace}}."',
```

Comment:
We should probably align to the error message formulation from the api-server
```suggestion
        'olm~User "{{user}}" cannot patch resource InstallPlans in API group "{{apiGroup}}" in the namespace "{{namespace}}."',
```
<img width="1755" alt="Screenshot 2021-11-18 at 23 21 03" src="https://user-images.githubusercontent.com/1668218/142506102-ea9e5b3b-ce54-4b9d-8b32-47c78b2dece4.png">

Reply:
Just checking with Megan to make sure she's cool with it!

Reply:
Hey @jhadvig - Megan was hoping to keep this message since it provides the "why" to the user. Making messaging clear to the users is a priority for the UXD team. If it's a goal to keep the UI and API messaging similar, would we be able to open a bug to add more details to the API message instead?

Reply:
@rebeccaalpert if this message if OK by the UXD team them Im fine ! 👍

Reply:
Okay cool; thank you! It means a lot to Megan.